internal/olm/operator/registry/configmap/configmap_test.go

```diff
@@ -0,0 +1,277 @@
+			userInput := []string{"userInput", "userInput2"}
+			b := make(map[string][]byte)
+			b, e := makeObjectBinaryData(obj, userInput...)
+
+			addObjectToBinaryData(binaryData, obj, userInput...)
+
+			Expect(e).Should(BeNil())
+			Expect(b).Should(Equal(binaryData))
+
+		})
+	})
+
+	Describe("makeBundleBinaryData", func() {
+		It("should serialize bundle to binary data", func() {
+			var e error
+			b := apimanifests.Bundle{
+				Name: "testbundle",
+				Objects: []*unstructured.Unstructured{
+					{
+						Object: map[string]interface{}{"val1": "val1"},
+					},
+					{
+						Object: map[string]interface{}{"val2": "va2"},
+					},
+				},
+			}
+
+			binaryData, err := makeBundleBinaryData(&b)
+			val := make(map[string][]byte)
+			for _, obj := range b.Objects {
+				e = addObjectToBinaryData(val, obj, obj.GetName(), obj.GetKind())
+			}
+
+			Expect(binaryData).Should(Equal(val))
+			Expect(e).Should(BeNil())
+			Expect(err).Should(BeNil())
+
+		})
+	})
+
+	Describe("makeConfigMapsForPackageManifests", func() {
+		It("should serialize packagemanifest to binary data", func() {
+			var e error
+			b := []*apimanifests.Bundle{
+				{
+					Name: "testbundle",
+					Objects: []*unstructured.Unstructured{
+						{
+							Object: map[string]interface{}{"val1": "val1"},
+						},
+						{
+							Object: map[string]interface{}{"val2": "va2"},
+						},
+					},
+					CSV: &v1alpha1.ClusterServiceVersion{
+						Spec: v1alpha1.ClusterServiceVersionSpec{
+							Version: version.OperatorVersion{
+								Version: semver.SpecVersion,
+							},
+						},
+					},
+				},
+				{
+					Name: "testbundle_2",
+					Objects: []*unstructured.Unstructured{
+						{
+							Object: map[string]interface{}{"val1": "val1"},
+						},
+						{
+							Object: map[string]interface{}{"val2": "va2"},
+						},
+					},
+					CSV: &v1alpha1.ClusterServiceVersion{
+						Spec: v1alpha1.ClusterServiceVersionSpec{
+							Version: version.OperatorVersion{
+								Version: semver.SpecVersion,
+							},
+						},
+					},
+				},
+			}
+
+			p := apimanifests.PackageManifest{
+				PackageName: "test_package_manifest",
+				Channels: []apimanifests.PackageChannel{
+					{Name: "test_1",
+						CurrentCSVName: "test_csv_1",
+					},
+					{Name: "test_2",
+						CurrentCSVName: "test_csv_2",
+					},
+				},
+				DefaultChannelName: "test_channel_name",
+			}
+
+			binaryDataByConfigMap, err := makeConfigMapsForPackageManifests(&p, b)
+
+			val := make(map[string]map[string][]byte)
+			cmName := getRegistryConfigMapName(p.PackageName) + "-package"
+			val[cmName], err = makeObjectBinaryData(p)
+			for _, bundle := range b {
+				version := bundle.CSV.Spec.Version.String()
+				e = fmt.Errorf("bundle ClusterServiceVersion %s has no version", bundle.CSV.GetName())
+
+				cmName := getRegistryConfigMapName(p.PackageName) + "-" + k8sutil.FormatOperatorNameDNS1123(version)
+				val[cmName], e = makeBundleBinaryData(bundle)
+			}
+
+			Expect(e).Should(BeNil())
```

Comment:
This `Expect(e)` needs to be in the for loop immediately after `makeBundleBinaryData` since you want the test to fail if any of the calls to it fail.